### PR TITLE
Update main example so that it can run smoothly

### PR DESCRIPTION
**Pull request by bagerard**
I recently looked into this project and was interested to play around with flask so I copy paste the main example but got an error since `get_random_pet` isn't defined. Perhaps it was done on purpose to avoid bloating the example with unnecessary details but I believe it would be worth providing an example that works out of the box

Next thing I updated is the endpoint spec, the response object requires a `description` to be valid (https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#response-object)

### CODE REVIEW

**Comment by lafrech** [comment]
> Perhaps it was done on purpose to avoid bloating the example with unnecessary details

Indeed, I'm pretty sure the `get_random_pet` was mean to be seen as a placeholder.

>  I believe it would be worth providing an example that works out of the box

I don't have a strong opinion but I'm not convinced about that. We could also make things more obvious by adding something like

    # Import model class
    from .model import Pet

and in the view func

    pet = Pet.random()

But current wording makes things obvious enough to me.

If we need an example that works out of the box, we may want to add an examples directory like in webargs.

I approve the other changes. Thanks for that.

We should also rename category into categories.

**Comment by bagerard** [comment]
Hi @lafrech , thanks for the review.
I've renamed "category" into "categories" and I also got rid of the `get_random_pet` function, I replaced it with a simple dict in the controller, IMO it doesn't complicate the code that much and makes it fully runnable. Let me know what you think of it, if you'r still not convinced, I'll switch it to 
```
from .model import Pet
...
pet = Pet.random()
```

**Comment by lafrech** [comment]
I'm fine with this change.

Nice reordering of the spec output. Perhaps you could use the same order on both examples (components before paths).

Also, please add yourself to AUTHORS.rst.

File: docs/index.rst [hunk 1]
```diff
@@ -57,12 +59,16 @@ Example Application
...
                 application/json:
                   schema: PetSchema
         """
-        pet = get_random_pet()
-        return PetSchema().dump(pet)
+        pet_data = {
+            "name": "sample_pet_" + str(uuid.uuid1()),
+            "categories": [{"id": 1, "name": "sample_category"}],
+        }
+        return PetSchema().dump(hardcoded_pet)
```
Comment (lafrech) [inline docs/index.rst]: Oops. Inconsistency.

Proposal:

```py
        # Hardcoded example data
        pet_data = {
            "name": "sample_pet_" + str(uuid.uuid1()),
            "categories": [{"id": 1, "name": "sample_category"}],
        }
        return PetSchema().dump(pet_data)
```

Comment (bagerard) [reply docs/index.rst]: damn... sorry for that. It should be fine now, I've re-tested it to make sure its running file

**Comment by lafrech** [comment]
Thanks.

**Pull request merged**